content/operate/rc/databases/create-database/create-active-active-database.md

```diff
@@ -98,10 +115,12 @@ The total operations per second:
 
 Throughput requirements grow dramatically as regions increase.  As a result, consider your requirements carefully.
 
+For Search and query databases, the estimated throughput from the [Sizing calculator](https://redis.io/redisearch-sizing-calculator/) is the total amount of additional throughput you need. When setting throughput for your Active-Active database, add this throughput divided across each region depending on your needs for each region.
+
```

Comment:
Should we consider a simple example here? Where the output from the calculator, for example 50000 is divided in 20000 writes(updates) and 30000 reads(queries)

Reply:
I added an example and clarified the throughput recommendations! Let me know what you think.